staging_service/utils.py

```diff
@@ -92,7 +91,6 @@ def __init__(self):
         """
         The ACLManager is used to add and remove acl endpoints for KBase Users on our Globus Share
         """
-        logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
```

Comment:
Moved to the start of app.py, since the new endpoint uses `logging` as well

Reply:
LGTM